lib/wakes/redirect_mapper.rb

```diff
@@ -1,28 +1,28 @@
 # frozen_string_literal: true
 module Wakes
   class RedirectMapper
-    attr_accessor :source, :target, :label
+    attr_accessor :source_host, :source_path, :target_host, :target_path, :label
 
-    def initialize(source, target, label = nil)
-      @source = source
-      @target = target
+    def initialize(source_path, target_path, label = nil, source_host = nil, target_host = nil)
+      @source_host = source_host
+      @source_path = source_path
+      @target_host = target_host
+      @target_path = target_path
       @label = label
-      puts Wakes.color(:white, "Redirecting from #{source} to #{target}", :bold => true)
 
+      puts Wakes.color(:white, "Redirecting from #{source_host}#{source_path} to #{target_host}#{target_path}",
+                       :bold => true)
       print_graph('Starting graph')
       both_present || target_present || source_present || none_present
-      print_graph('Ending graph')
-      puts ''
+      print_graph('Ending graph', :newline => true)
     end
 
     private
 
-    def print_graph(notice)
+    def print_graph(notice, options = {})
       puts Wakes.color(:magenta, notice)
-
-      Wakes::Resource.where(:id => resources_affected).each do |resource|
-        puts resource.to_s
-      end
+      Wakes::Resource.where(:id => resources_affected).each { |resource| puts resource.to_s }
+      puts '' if options[:newline]
```

Comment:
@benhutton I meant to discuss this with you. Rubocop was rejecting this class because it is too long, but I did not see an obvious place to trim it. This is admittedly a hack but it seemed this would be true anywhere I trimmed. 

My preference was actually to use parallel assignment [here](https://github.com/desiringgod/wakes/blob/01a02b9efa94278f5bbdc13da1615c7848b14ab2/lib/wakes/redirect_mapper.rb#L7-L10), but I see that is discouraged in the style guide and Rubocop. Alternatively, we could drop the [bold](https://github.com/desiringgod/wakes/blob/01a02b9efa94278f5bbdc13da1615c7848b14ab2/lib/wakes/redirect_mapper.rb#L14).

Open to suggestions here. I'm dancing for Rubocop but don't actually think the class needs refactoring.